CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -1636,6 +1636,26 @@ void arm_softmax_with_batch_q7(const q7_t * vec_in, const uint16_t nb_batches,co
 
 void arm_softmax_q15(const q15_t * vec_in, const uint16_t dim_vec, q15_t * p_out);
 
+  /**
+   * @brief S8 softmax function
+   * @param[in]  input     Pointer to the input tensor
+   * @param[in]  num_rows  Number of rows in the input tensor
+   * @param[in]  row_size  Number of elements in each input row
+   * @param[in]  mult      Input quantization multiplier
+   * @param[in]  shift     Input quantization shift
+   * @param[in]  diff_min  Minimum difference with max in row to perform op
+   * @param[out] output    Pointer to the output tensor
+   *
+   */
+
+void arm_softmax_s8(const int8_t *input,
```

Comment:
I would rename it to arm_softmax_1d_s8 since we only target the 1d case with this function. This aspect should be also reported in the doxygen comment